### PR TITLE
Customize alert panel

### PR DESCRIPTION
**Pull request by agenius-shannon**
https://3.basecamp.com/3775257/buckets/16956142/todos/6558707171

### CODE REVIEW

**Review by cb4ds — APPROVED** [review]

**Comment by agenius-mohammed-ali** [comment]
> ## [Codecov](https://app.codecov.io/gh/Aggregate-Genius/periscope2/pull/51?src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Aggregate-Genius) Report
> All modified and coverable lines are covered by tests ✅
> 
> > Comparison is base [(`ddebe11`)](https://app.codecov.io/gh/Aggregate-Genius/periscope2/commit/ddebe11fbcf48812d7b89ccac53d7ea987758783?el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Aggregate-Genius) 93.80% compared to head [(`d8cfc3c`)](https://app.codecov.io/gh/Aggregate-Genius/periscope2/pull/51?src=pr&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Aggregate-Genius) 93.59%.
> 
> Additional details and impacted files
> [☔ View full report in Codecov by Sentry](https://app.codecov.io/gh/Aggregate-Genius/periscope2/pull/51?src=pr&el=continue&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Aggregate-Genius). 📢 Have feedback on the report? [Share it here](https://about.codecov.io/codecov-pr-comment-feedback/?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=Aggregate-Genius).

Unit test coverage decreased. This can resolved by covering all cases for `createPSAlert` method unit tests. Check https://app.codecov.io/gh/Aggregate-Genius/periscope2/pull/51/blob/R/ui_helpers.R for more details

File: R/fw_helpers_external.R [hunk 1]
```diff
@@ -326,7 +326,7 @@ load_announcements <- function() {
             }
 
             if (valid) {
-                createAlert(id      = "announceAlert",
+                createPSAlert(id      = "announceAlert",
                             options = list(title    = title,
```
Comment (agenius-mohammed-ali) [inline R/fw_helpers_external.R:330]: indentation

File: R/ui_helpers.R [hunk 2]
```diff
@@ -359,7 +359,7 @@ add_ui_body <- function(body_elements = NULL, append = FALSE) {
 #' @seealso \link[periscope2:get_url_parameters]{periscope2:get_url_parameters()}
 #'
 #' @export
-createAlert <- function(session  = shiny::getDefaultReactiveDomain(),
+createPSAlert <- function(session  = shiny::getDefaultReactiveDomain(),
```
Comment (agenius-mohammed-ali) [inline R/ui_helpers.R:362]: indentation

File: tests/testthat/sample_app_left_sidebar/program/ui_body.R [hunk 3]
```diff
@@ -341,7 +341,7 @@ alerts_box <- box(
     collapsible = TRUE,
     width       = 12,
     p("Alerts can be added with the following code in the server:"),
-    blockQuote(tags$pre("S:periscope2::createAlert(id = 'alert place',
+    blockQuote(tags$pre("S:createPSAlert(id = 'alert place',
```
Comment (agenius-mohammed-ali) [inline tests/testthat/sample_app_left_sidebar/program/ui_body.R:344]: indentation

File: R/fw_helpers_external.R (hunk 1 above)
Comment (agenius-mohammed-ali) [reply R/fw_helpers_external.R:330]: done

File: R/ui_helpers.R (hunk 2 above)
Comment (agenius-mohammed-ali) [reply R/ui_helpers.R:362]: done

File: tests/testthat/sample_app_left_sidebar/program/ui_body.R (hunk 3 above)
Comment (agenius-mohammed-ali) [reply tests/testthat/sample_app_left_sidebar/program/ui_body.R:344]: done

**Pull request merged**